App.js

```diff
@@ -1,14 +1,15 @@
-import React, { useEffect, useState, useCallback } from 'react'
+import React, { useCallback,useEffect, useState } from 'react'
```

Comment:
maybe in that sense we could add a rule for space after comma?

Reply:
yeah, that's unfortunately an oddity in the eslint-simple-import-sort package which can't be fixed. We could check if there's an eslint rule, however if you use Prettier to format your code it will fix it as well